.swiftlint.yml

```diff
@@ -77,6 +77,9 @@ opt_in_rules:
   - xct_specific_matcher
   - yoda_condition
 
+attributes:
+  always_on_line_above:
+    - "@OptionGroup"
```

Comment:
Opinions on that? I think the list of command line arguments looks better if attributes are all above the variable declarations.

Reply:
Fine for you, @jpsim?